src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/list/main.js

```diff
@@ -37,7 +37,16 @@ define(function () {
                 toolbar: {
                     buttons: {
                         add: {},
-                        deleteSelected: {}
+                        deleteSelected: {},
+                        export: {
+                            options: {
+                                urlParameter: {
+                                    flat: true,
+                                    limit: 999999
```

Comment:
the problem here is, that the rest-endpoint is the same as for the list. therefor we have to fix it there but currently i would add this attribute or should we provide a default value for that?